meta/contributor-handbook.md

```diff
@@ -0,0 +1,89 @@
+| 1 | Initial | Contains an initial pattern idea or what we call a donut (a pattern with missing sections). | Not included in any publications | A few days / weeks |
+| 2 | Structured | Contains a complete pattern that it is not properly validated yet (e.g. because it only synthesizes an idea or the experiences from one instead of three organizations). | Included in the patterns catalogue but with a warning | A few days / weeks |
+| 3 | Validated | The pattern is validated (e.g. because three or more instances exists and are synthesized by the pattern) and its writing is of high quality. | Included in the patterns catalogue | One or two months |
+| 4 | Established | The pattern is validated, its writing is of "text book" quality, and the community agrees it is responsible to use it as teaching material. | Included in the patterns catalogue and patterns book | As long as it takes |
+
+For the first pattern you contribute, you should aim for maturity levels `1: initial` or `2: structured`. 
+
+If you want to help promoting a pattern one maturity level up, we suggest to first create an issue to discuss the matter and see if someone else is working on that already. After that, you can create a pull request with the necessary changes.
+
+To achieve a given maturity level a pattern has to satisfy the requirements for that given maturity level and lower levels. The following sections lay out the requirements per maturity level in detail.
+
+
+### Requirements: Level 1 - Initial
+
+Patterns (or other documents) of level 1 are stored in the directory `/1-initial`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	- Donuts - patterns without solution or other missing sections (`donut-*.md`)
+	- Tale - unstructured information in any form (`tale-*.md`)
+	
+- Validation requirements:
+	- N/A
+	
+- Content requirements:
+	- The document is readible & comprehensible for other's than the author(s)
+	- The author(s) contribute the contents according to the [license](../LICENSE.txt) & are allowed to do so
+	- Thoughts and contents by third parties are quoted / referenced explicitly
+
+	
+### Requirements: Level 2 - Structured
+
+Patterns of level 2 are stored in the directory `/2-structured`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Is partially validated by at least one known instance
+		- Or: pattern warrants to still be included (e.g. because validation is not feasible in the foreseeable future)
+	
+- Content requirements:
+	- Complies with the [patterns format](pattern-template.md)
+	- Complies with [basic style guide](#) - *Oops! We have not yet developed this*
+	
+	
+### Requirements: Level 3 - Agreed
+
+Patterns of level 3 are stored in the directory `/3-agreed`.
+
+- Document types:
+	- Patterns (`pattern-*.md`)
+	
+- Validation requirements:
+	- Is validated by at least three known instances
+		- Or: pattern warrants to still be included (e.g. because validation is not feasible in the foreseeable future)

```

Comment:
Suggest deleting here--this doesn't make sense if the pattern already has three known instances validating it.

Reply:
Good point! Could you raise a commit suggestion for that please @NewMexicoKid?

Reply:
The following concensus seems to be forming more and more: Don't open loopholes in regards to validation. I think in the long run, this will pay off :) I happily deleted this line and will mark this thread as resolved.